webapp/channels/src/packages/mattermost-redux/src/actions/general.ts

```diff
@@ -27,6 +27,8 @@ export function getClientConfig(): ActionFuncAsync<ClientConfig> {
         Client4.setDiagnosticId(data.DiagnosticId);
 
         dispatch({type: GeneralTypes.CLIENT_CONFIG_RECEIVED, data});
+        const type = data.AppsPluginEnabled === 'true' ? AppsTypes.APPS_PLUGIN_ENABLED : AppsTypes.APPS_PLUGIN_DISABLED;
+        dispatch({type});
```

Comment:
Small change here. Since these are both non-thunk actions, you can combine the `CLIENT_CONFIG_RECEIVED` and `APPS_PLUGIN_ENABLED`/`APPS_PLUGIN_DISABLED` into a single action with `batchActions` to make it so that the UI only re-renders once for the Redux changes for both. That's not always that important, but it'd be good to do here since this is part of the initial load which already causes the app to re-render a lot

Reply:
Done!